.gitmodules

```diff
@@ -1,7 +1,7 @@
 [submodule "vendored/puppet"]
 	path = vendored/puppet
 	url = https://github.com/puppetlabs/puppet
-	branch = master
+	branch = apply-grammar
```

Comment:
@nicklewis pointed out that this grammar currently allows resource declarations in the arguments that would not normally be allowed in a plan. Need to fix that.

Reply:
Fixed.